apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.component.ts

```diff
@@ -0,0 +1,78 @@
+import { DynamicConfigurable } from '@o3r/configuration';
+import { COMPONENT_REPLACEMENT_PRES_DEFAULT_CONFIG } from './component-replacement-pres.config';
+import { COMPONENT_REPLACEMENT_PRES_CONFIG_ID } from './component-replacement-pres.config';
+import { ComponentReplacementPresConfig } from './component-replacement-pres.config';
+import { Observable } from 'rxjs';
+import { DatePickerInputPresComponent } from '../../utilities/date-picker-input/date-picker-input-pres.component';
+import { Configuration, Functionify } from '@o3r/core';
+import { DatePickerInputPresContextInput, DatePickerInputPresContextOutput } from '../../utilities/date-picker-input/date-picker-input-pres.context';
+import { C11nModule, C11nService } from '@o3r/components';
+import { CommonModule, formatDate } from '@angular/common';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+
+const ONE_DAY_IN_MS = 24 * 60 * 60 * 1000;
+
+@O3rComponent({ componentType: 'ExposedComponent' })
+@Component({
+  selector: 'o3r-component-replacement-pres',
+  standalone: true,
+  imports: [C11nModule, CommonModule, ReactiveFormsModule],
+  templateUrl: './component-replacement-pres.template.html',
+  styleUrl: './component-replacement-pres.style.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ComponentReplacementPresComponent implements OnChanges, DynamicConfigurable<ComponentReplacementPresConfig> {
+
+  /** Configuration stream based on the input and the stored configuration*/
+  public config$: Observable<ComponentReplacementPresConfig>;
+
+  @ConfigObserver()
+  private readonly dynamicConfig$: ConfigurationObserver<ComponentReplacementPresConfig>;
+
+  /** Input configuration to override the default configuration of the component*/
+  @Input()
+  public config: Partial<ComponentReplacementPresConfig> | undefined;
+
+  /** Observable of the presenter that we want to use, processed by the c11n directive */
+  public presenter$!: Observable<Context<DatePickerInputPresContextInput, DatePickerInputPresContextOutput> & DynamicConfigurable<Configuration>>;
+
+  /** Convenience object to prepare all the outputs binding in advance */
+  public outputs!: Functionify<DatePickerInputPresContextOutput>;
+
+  public initialDate = this.formatDate(Date.now() + 7 * ONE_DAY_IN_MS);
+
+  public dateFormControl = new FormControl<string | null>(this.initialDate);
+
+  constructor(
+    private readonly c11nService: C11nService,
+    @Optional() configurationService: ConfigurationBaseService) {
+    this.dynamicConfig$ = new ConfigurationObserver<ComponentReplacementPresConfig>(COMPONENT_REPLACEMENT_PRES_CONFIG_ID, COMPONENT_REPLACEMENT_PRES_DEFAULT_CONFIG, configurationService);
+    this.config$ = this.dynamicConfig$.asObservable();
+    this.loadPresenter();
+    this.dynamicConfig$.next({ ...this.config, datePickerCustomKey: 'exampleDatePickerFlavorHebrew' });
```

Comment:
Maybe we can have a button in the page to trigger this change 
So user can easily see the 2 flavors

Reply:
And so it will go through the store as it will work in real use case

Reply:
Indeed that's the final goal. But this issue is blocking it. When solved, we'll have the implementation of the switch button. 
https://github.com/AmadeusITGroup/otter/issues/1168